src/saucelabs/job.js

```diff
@@ -132,12 +135,11 @@ export default class Job {
             name:             this.options.testName,
             tags:             this.options.tags,
             build:            this.options.build,
-            platform:         this.platform[0],
-            browserName:      this.platform[1],
-            version:          this.platform[2],
             tunnelIdentifier: this.options.tunnelIdentifier
         };
 
+        assign(this.browserInfo, initBrowserParams);
```

Comment:
Should copy the info from this.browserInfo to initBrowserParams. So it's
```js
 assign(initBrowserParams, this.browserInfo);
```

Because we should not change `this.browserInfo`